how_to_build.md

```diff
@@ -41,7 +43,7 @@ Using a command prompt, navigate to your clone root folder and execute:
 
 `bundle exec rake`
 
-This executes the default build tasks. After the build has completed, the build artifacts will be located in `Build`.

```

Comment:
About this, I think TeamCity is still looking for artifacts in `Build` (in addition to `artifacts`). I noticed it a few days ago but didn't have time to check if it was normal; I don't think it is.